### PR TITLE
Cleanup some Peep/Guest/Staff code

### PR DESCRIPTION
**Pull request by ZehMatt**

### CODE REVIEW

File: src/openrct2/peep/Peep.cpp [hunk 1]
```diff
@@ -641,15 +643,13 @@ std::optional<CoordsXY> Peep::UpdateAction(int16_t& xy_distance)
         action_sprite_image_offset = 0;
         action = PEEP_ACTION_NONE_2;
         UpdateCurrentActionSpriteType();
-        Invalidate();
         return { { x, y } };
     }
     action_sprite_image_offset = peepAnimation[action_sprite_type].frame_offsets[action_frame];
```
Comment (duncanspumpkin) [inline src/openrct2/peep/Peep.cpp]: Are you sure this is sensible. Peeps don't always move when the action is being updated so wouldn't always call invalidate.

Comment (ZehMatt) [reply src/openrct2/peep/Peep.cpp]: UpdateCurrentActionSpriteType will call Invalidate

Comment (duncanspumpkin) [reply src/openrct2/peep/Peep.cpp]: But UpdateCurrentActionSpriteType isn't called in this branch.

Comment (ZehMatt) [reply src/openrct2/peep/Peep.cpp]: Yes but it returns a location and calls MoveTo which then invalidates. I haven't spotted any drawing glitches so far but would be nice if you gave it a spin as well.

**Comment by ZehMatt** [comment]
I've added Invalidate calls to where the return value is ignored, it should cover everything now. It might still call it twice here and there but now we have less code.

**Pull request merged**